src/applications/vaos/new-appointment/components/ReasonForAppointmentPage.jsx

```diff
@@ -66,6 +70,7 @@ const pageKey = 'reasonForAppointment';
 
 export default function ReasonForAppointmentPage() {
   const pageTitle = useSelector(state => getPageTitle(state, pageKey));
+  const flowType = useSelector(state => getFlowType(state));
```

Comment:
Nit: Could change to this since the only param needed for the selector is 'state'.
```
const flowType = useSelector(getFlowType);
```